src/main/java/com/amadeus/resources/DirectDestination.java

```diff
@@ -0,0 +1,20 @@
+package com.amadeus.resources;
+
+import lombok.Getter;
+import lombok.ToString;
+
+/**
+ * An DirectDestination object as returned by the DirectDestinations API.
+ * @see com.amadeus.airport.DirectDestinations#get()
+ */
+
+@ToString
+public class DirectDestination extends Resource {
```

Comment:
As a suggestion, you could rename the `DirectDestination` to `Destination`, If I am correct that's the naming in the model.

Reply:
> As a suggestion, you could rename the `DirectDestination` to `Destination`, If I am correct that's the naming in the model.

@tsolakoua Make sense and I agree! But I still feel a little bit confused about the way of naming.
So here is my understanding:
For the classes in the resources package, they should follow the naming in the model, which should be **Destination** in this case.
For the classes in the different function package(eg. _com.amadeus.airport_), they should follow the naming in the API endpoint, which should be **DirectDestination** in this case. 
Could you help me check if I get the correct understanding? 
Thank you very much!

Reply:
For the functions to call each API:
- The SDK maps every API path to a similar path. For example, `/v1/airport/direct-destinations` is the SDK is represented as`amadeus.airport.directDestinations`. So the classes `Airport` and `DirectDestinations` are indeed needed as you have already implemented them.

For the names of the resource files:
- Let's use for each resource file the name of the model. If the model is Destination the resource file should be `Destination.java`. There might be some inconsistencies from past implementations but from now let's try to keep this convention.  

Hope that helps @xianqiliu :) Let me know if you have more questions.